src/zepto.js

```diff
@@ -678,6 +678,8 @@ var Zepto = (function() {
         $this.css(props)
       })
       if (!this.length) return null
+      if (!Zepto.contains(document.documentElement, this[0]))
```

Comment:
Thanks for the fix. To avoid the extra check, couldn't we just wrap the `getBoundingClientRect()` in a `try...catch` block?